Formula/bookkeeper.rb

```diff
@@ -0,0 +1,61 @@
+class Bookkeeper < Formula
+  desc "Replicated log service for building replicated state machines"
+  homepage "https://bookkeeper.apache.org/"
+  url "https://apache.claz.org/bookkeeper/bookkeeper-4.5.0/bookkeeper-server-4.5.0-bin.tar.gz"
```

Comment:
Use the generic Apache mirror rather than hardcoding a specific one, like:
```ruby
url "https://www.apache.org/dyn/closer.cgi?path=drill/drill-1.10.0/apache-drill-1.10.0.tar.gz"
```